app/src/components/ResetButton.tsx

```diff
@@ -0,0 +1,7 @@
+import MainButton from './MainButton.tsx'
```

Comment:
Same thing as the other PR, I can't see the MainButton in this branch.
It should have those commits if you need them here.